csrankings.csv

```diff
@@ -16396,8 +16395,10 @@ Qingcai chen,Harbin Institute of Technology,http://www.hitsz.edu.cn/teacher/view
 Buzou Tang,Harbin Institute of Technology,http://www.hitsz.edu.cn/teacher/view/id-1166.html,5oGXsxUAAAAJ
 Bingquan Liu,Harbin Institute of Technology,http://homepage.hit.edu.cn/liubingquan,NOSCHOLARPAGE
 Yuanchao Liu,Harbin Institute of Technology,http://homepage.hit.edu.cn/ycliu,NOSCHOLARPAGE
-Chengjie Sun,Harbin Institute of Technology,http://homepage.hit.edu.cn/sunchengjie,NOSCHOLARPAGE
Lili Shan,Harbin Institute of Technology,http://homepage.hit.edu.cn/shanlili,8NObiJAAAAAJ
+Lili Shan,Harbin Institute of Technology,http://homepage.hit.edu.cn/shanlili,8NObiJAAAAAJ
 Tiejun Zhao,Harbin Institute of Technology,http://homepage.hit.edu.cn/zhaotiejun,NOSCHOLARPAGE
 Muyun  Yang,Harbin Institute of Technology,http://mitlab.hit.edu.cn/2018/0608/c9183a210161/page.htm,NOSCHOLARPAGE
 Hailong Cao,Harbin Institute of Technology,http://mitlab.hit.edu.cn/2018/0608/c9183a210165/page.htm,NOSCHOLARPAGE
 Conghui Zhu,Harbin Institute of Technology,http://homepage.hit.edu.cn/conghui,NOSCHOLARPAGE
+Yanyan Zhao,Harbin Institute of Technology,http://homepage.hit.edu.cn/yanyan,NOSCHOLARPAGE
```

Comment:
Does not appear to be in the CS department.

Reply:
It's my mistake, sorry. I have removed Yanyan Zhao from the file.